tests/controller-test.js

```diff
@@ -212,7 +212,9 @@ OO = {
       createPluginElements: function() {},
       findMainVideoElement: function(a) {},
       loadConfigData: function(a, b, c, d) {},
-      cleanUpEventListeners: function(){}
+      cleanUpEventListeners: function(){},
```

Comment:
Please make the spacing consistent with all the other function declarations here.